bookshelf/src/main/java/com/example/managedvms/gettingstartedjava/basicactions/CreateBookServlet.java

```diff
@@ -46,9 +44,6 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException,
       IOException {
-    req.setAttribute(
```

Comment:
I moved it so that session variables could be set in a central location. Also, since the setting doesn't change throughout the life of the app, I thought it made sense to set it once instead of checking with every new request